workspaces/azure-devops/.changeset/lemon-geese-pump.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage-community/plugin-scaffolder-backend-module-azure-pipeline-actions': minor
```

Comment:
```suggestion
'@backstage-community/plugin-scaffolder-backend-module-azure-devops': patch
```

How does that sound for naming? I'd like to have all the action in one module. With the module in the Backstage repo currently called `@backstage/plugin-scaffolder-backend-module-azure` this seems like a nice focused name.

Reply:
Sounds great!